src/change-hub.js

```diff
@@ -29,7 +29,7 @@ function loadRoomObjects(hubId) {
   objectsScene.appendChild(objectsEl);
 }
 
-export async function changeHub(hubId, addToHistory = true, waypoint = null) {
+export async function changeHub(hubId, addToHistory = true, waypoint = "") {
```

Comment:
What is this change for?

Reply:
If I remember correctly types that can be recognized by typescript compiler can be specified even in JavaScript with @types. But I think defaulting with "" is also ok (and probably it's simpler).